mp-backend/db-tables.yml

```diff
@@ -22,6 +22,33 @@ authTable:
       ProvisionedThroughput:
           ReadCapacityUnits: 1
           WriteCapacityUnits: 1
+    ProvisionedThroughput:
+      ReadCapacityUnits: 1
+      WriteCapacityUnits: 1
+recipeTable:
+  Type: AWS::DynamoDB::Table
+  Properties:
+    TableName: recipe
+    AttributeDefinitions:
+      - AttributeName: id
+        AttributeType: S
+    KeySchema:
+      - AttributeName: id
+        KeyType: HASH
+    GlobalSecondaryIndexes:
+    - IndexName: IdToRecipe
```

Comment:
Global Secondary Indices are used to lookup information by an attribute that isn't defined in the KeySchema. I think the intention here was to be able to lookup from a UserId, the list of recipes that the user has. I think we should just enforce that our recipe schema always has a recipeId and a userId as mentioned above. While a GSI would work just fine, it feels more natural to be able to sort the data based on ID, instead of doing a reverse lookup.